framework/src/base/MooseObject.C

```diff
@@ -117,7 +128,10 @@ MooseObject::getDataFileName(const std::string & param)
   {
     const auto path = MooseUtils::pathjoin(data_dir, relative_path);
     if (MooseUtils::checkFileReadable(path, false, false, false))
+    {
+      mooseInfo("Data file '", path, "' found in a source repository.");
```

Comment:
paramInfo?

Reply:
hm... then I'd have to pass the parameter in. That's only available in the other method.